api/parts/mgmt/ip.js

```diff
@@ -0,0 +1,47 @@
+var ip = {},
```

Comment:
Since it is a new utility module, please comment it in JS Doc format, so others can learn about it and know how to use it. Here is an example:
https://github.com/Countly/countly-server/blob/master/api/utils/authorizer.js